src/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+set(Montage_SRC
+  itkParseTileConfiguration.cxx
+  )
+
+add_library(Montage ${Montage_SRC})
+# target_link_libraries(Montage ${ITKCommon}) # we do not link-depend on anything
+itk_module_target(Montage)
```

Comment:
here we want to use 

```
itk_module_add_library(Montage ${Montage_SRC})
```

per

https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/src/CMakeLists.txt